app/src/main/java/com/marverenic/music/player/PlayerController.java

```diff
@@ -18,6 +18,10 @@
  */
 public interface PlayerController {
 
+    int MINIMUM_SONG_PER_CHUNK = 500;
```

Comment:
Isn't this a maximum? I think it'd make more sense to be called `MAXIMUM_CHUNK_ENTRIES` or something.